### ISSUE TITLE
Devops - Doc a11y scanner failing

### ISSUE THREAD

**Issue by jbergman-oddball** [department-of-veterans-affairs/va-mobile-app#12553]
### Type of change

Fix Automation Bug

### What is changing

Github Actions

### Work Being Done

The docs scanner is failing due to a version mismatch.  
```bash
0s
Run # Path to the downloaded sitemap file
Error: session not created: This version of ChromeDriver only supports Chrome version 145
Current browser version is 144.0.7559.132 with binary path /opt/google/chrome/chrome
Error: Process completed with exit code 2.
```

### Acceptance Criteria

- [ ] error is resolved
- [ ] code is updated
